src/main/java/uk/gov/hmcts/reform/civil/service/search/TrialReadyCheckSearchService.java

```diff
@@ -31,6 +31,7 @@ public Query query(int startIndex) {
                                                                             .atTime(LocalTime.MIN).plusWeeks(3)
                                                                             .toString()))
                             .must(beState(PREPARE_FOR_HEARING_CONDUCT_HEARING))
+                            .mustNot(matchQuery("data.allocatedTrack", "SMALL_CLAIM"))
```

Comment:
Can be replaced with the enum constant AllocatedTrack.SMALL_CLAIM.toString() value. It would avoid having a hard-coded string.